### ISSUE TITLE
missing template file in wagtailadmin 4.0.x is breaking wagtailmenus

### ISSUE THREAD

**Issue by jcuotpc** [jazzband/wagtailmenus#433]
I get the following error after updating to wagtail 4.0.1 or 4.0.2
`django.template.exceptions.TemplateDoesNotExist: wagtailadmin/pages/_editor_css.html`

I checked the wagtail repository and the `_editor_css.html` file is missing.

I believe one "include" line needs to be removed from `/wagtailmenus/templates/wagtailmenus/mainmenu_edit.html`

```{% extends "modeladmin/create.html" %}

{% block extra_css %}
<!--    {% include "wagtailadmin/pages/_editor_css.html" %}-->
    {{ edit_handler.form.media.css }}
    {{ view.media.css }}
{% endblock %}

{% block extra_js %}
    {% include "wagtailadmin/pages/_editor_js.html" %}
    {{ edit_handler.form.media.js }}
    {{ edit_handler.html_declarations }}
    {{ view.media.js }}
{% endblock %}

{% block header %}
    {% include "wagtailmenus/includes/header_with_site_switcher.html" with title=view.page_title subtitle=view.get_page_subtitle icon=view.header_icon %}
{% endblock %}

{% block form_action %}?{% endblock %}
```

**Comment by saevarom** [comment]
I believe this is solved by #432

**Comment by jcuotpc** [comment]
It works fine now. Thank you for updating this very useful package!
